x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/flyout/fields_table.tsx

```diff
@@ -44,35 +55,54 @@ export const IndicatorFieldsTable: FC<IndicatorFieldsTableProps> = ({
               defaultMessage="Value"
             />
           ),
-          render: (field: string) => <IndicatorFieldValue indicator={indicator} field={field} />,
+          render: (item: TableItem) => (
+            <SecurityCellActions
+              data={{ field: item.key, value: item.value }}
+              mode={CellActionsMode.HOVER_DOWN}
+              triggerId={SecurityCellActionsTrigger.DEFAULT}
+            >
+              <IndicatorFieldValue indicator={indicator} field={item.key} />
+            </SecurityCellActions>
+          ),
         },
+      ] as Array<EuiBasicTableColumn<TableItem>>,
+    [indicator]
+  );
+
+  const search = useMemo(
+    () => ({
+      box: {
+        schema: true,
+        incremental: true,
+      },
+    }),
+    []
+  );
+
+  const items = useMemo(() => {
+    return fields.toSorted().reduce<TableItem[]>((acc, field) => {
+      return [
+        ...acc,
         {
-          actions: [
-            {
-              render: (field: string) => (
-                <IndicatorValueActions
```

Comment:
Now that we're not using `IndicatorValueActions` here anymore, it could be worth trying to remove its usage from [the Block component](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/flyout/block.tsx) and get rid of the `IndicatorValueActions` component entirely?